modules/microshift-firewall-update-for-service.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-firewall.adoc
+
+
+:_content-type: CONCEPT
+[id="microshift-firewall-update-for-service_{context}"]
+= Updating firewall ports when a service is exposed
+
+Firewalld is often active when you run services on {microshift-short}. This can disrupt certain services on {microshift-short} because traffic to the ports might be blocked by the firewall. You need to ensure the necessary firewall ports are open if you want certain services to be accessible from outside the host. There are several options for opening your ports:
+
+* Services of the `NodePort` and `LoadBalancer` type are automatically available with OVN-K.
++
+In these cases, OVN-K adds iptables rules so the traffic to the node IP address is delivered to the relevant ports. This is done using the PREROUTING rule chain and is then forwarded to the OVN-K to bypass the firewalld rules for local host ports and services. Iptables and firewalld are backed by nftables in RHEL9. The nftables rules, that the iptables generates, always has priority over the rules that the firewalld generates.
+
+* Pods with the `HostPort` parameter settings are automatically available. This also includes the router-default pod, which uses ports 80 and 443.
++
+For `HostPort` pods, the CRI-O config sets up iptables DNAT rules to the pods IP address and port.
+
+These methods function for clients on the same host or remote host. The iptables rules, that are added by OVN-K and CRI-O, attach to the PREROUTING and OUTPUT chains. The local traffic goes through the OUTPUT chain with the interface set to the `lo` type. The DNAT runs before it hits filler rules in the INPUT chain.
```

Comment:
* Should some of these items be in backticks?
* Same question about "OVN-K"
* Should people know what DNAT is, or does it need spelled out on first use above?
* A few wording suggestions, as long as it doesn't change the meaning

```suggestion
These methods function for clients whether they are on the same host or on a remote host. The iptables rules, which are added by OVN-K and CRI-O, attach to the `PREROUTING` and `OUTPUT` chains. The local traffic goes through the `OUTPUT` chain with the interface set to the `lo` type. The DNAT runs before it hits filler rules in the `INPUT` chain.
```